epub33/core/index.html

```diff
@@ -1191,6 +1205,14 @@ <h4>Package Document Definition</h4>
 					<p>All [[XML]] elements defined in this section are in the <code>http://www.idpf.org/2007/opf</code>
 						namespace [[XML-NAMES]] unless otherwise specified.</p>
 
+					<section>
+						<h4>Parsing URLs in the Package Document</h4>
+						<p>
+							To parse a URL string <var>url</var> used in the Package Document, the <a data-cite="url#concept-url-parser">URL Parser</a> [[URL]] MUST be applied to <var>url</var>, with the 
+							<a>container root URL</a> as <var>base</var>.
```

Comment:
OMG I'm just noticing this is breaking EPUB 😱

URLs in the package document must be parsed with the content URL of the Package Document as base, not the container root URL. 🤦‍♂️

(btw, "content URL" sound a bit odd when talking about the URL of the Package Document, but let's not go down another round of naming debate 😅).

Reply:
Ya, but wouldn't this only require a separate statement on obtaining the Package Document URL first? Something like:

> To parse relative URL strings in the Package Document requires first obtaining the URL of the Package Document itself and then using that URL to resolve the relative strings.
> 
> To obtain the Package Document URL, the URL Parser [[URL]] MUST be applied to the path to the Package Document specified in the `container.xml` file with the container root URL as `base`.
> 
> To parse any relative URL string `url` in the Package Document, the URL Parser [[URL]] MUST be applied to `url` with the Package Document URL as `base`.

Reply:
> wouldn't this only require a separate statement on obtaining the Package Document URL first

Not even! fortunately "content URL" is well-defined now, the "content URL of the Package Document" is already defined.

The change should be as simple as s/container root URL/content URL of the Package Document/

Reply:
Oops indeed... This is what happens with a complex change in the document :-)

Anyway. Settled this, too, in the upcoming commit.

Reply:
Ah, I didn't see that in 6.1.5.

You admittedly lost me with what you were saying, as I ended up click those stale links again and didn't even find the term in the spec.

There's a weird lack of symmetry between container rool url and content url, plus the latter sounds like something you'd find in content documents rather than paths to things in the abstract container, which may only be directories and not "content".